index.bs

```diff
@@ -2829,31 +2898,64 @@ an assertion. Its {{PublicKeyCredentialRequestOptions/challenge}} member MUST be
+
+            The [=list/items=] SHOULD specify {{PublicKeyCredentialDescriptor/transports}} whenever possible.
+            This helps the [=client=] optimize the user experience for any given situation.
+            Also note that the [=[RP]=] does not need to filter the list when requesting [=user verification=] &mdash;
+            the [=client=] will automatically ignore non-eligible credentials
+            if {{PublicKeyCredentialRequestOptions/userVerification}} is set to {{UserVerificationRequirement/required}}.
+
+          - If the [=user account=] to authenticate is not already identified,
+            then the [=[RP]=] MAY leave this member [=list/empty=] or unspecified.
+            In this case, only [=discoverable credentials=] will be utilized in this [=authentication ceremony=],
+            and the [=user account=] MAY be identified by the {{AuthenticatorAssertionResponse/userHandle}}
+            of the resulting {{AuthenticatorAssertionResponse}}.
+            If the available [=authenticators=] [=contain=] more than one [=discoverable credential=] [=scoped=] to the [=[RP]=],
+            the credentials are displayed for the user to select from (see step 7 of [[#sctn-op-get-assertion]]).
+
+        If not [=list/empty=], the client MUST return an error if none of the listed credentials can be used.
+
+        The list is ordered in descending order of preference: the first item in the list is the most
+        preferred credential, and the last is the least preferred.
 
     :   <dfn>userVerification</dfn>
-    ::  This OPTIONAL member describes the [=[RP]=]'s requirements regarding [=user verification=] for the
+    ::  This OPTIONAL member specifies the [=[RP]=]'s requirements regarding [=user verification=] for the
         {{CredentialsContainer/get()}} operation. The value SHOULD be a member of {{UserVerificationRequirement}} but [=client platforms=] MUST ignore unknown values, treating an unknown value as if the [=map/exist|member does not exist=]. Eligible authenticators are filtered to only those capable of satisfying this requirement.
 
+        See {{UserVerificationRequirement}} for the description of {{AuthenticatorSelectionCriteria/userVerification}}'s values and semantics.
+
     :   <dfn>extensions</dfn>
-    ::  This OPTIONAL member contains additional parameters requesting additional processing by the client and authenticator.
-        For example, if transaction confirmation is sought from the user, then the prompt string might be included as an
-        extension.
+    ::  This OPTIONAL member contains [=client extension inputs=] requesting additional processing by the [=client=] and [=authenticator=].
+        For example, the [=[RP]=] may request that the authenticator returns additional information
+        about what kind of [=user verification=] was performed.
+        Some extensions are defined in [[#sctn-extensions]];
```

Comment:
Good point, and good catch! Maybe `credProps` is a better example?

Reply:
Ah wait, this occurrence is in getAssertion, and `credProps` is in fact the example in the corresponding text for makeCredential. As far as I know the only widely implemented assertion extension is `appid`, which I also don't think makes for a great example since it's a backwards compatibility hook rather than a "feature" in its own right. So... do we drop the example? Keep it? Change it?

Reply:
Ah, look at me not being able to read your whole comment... :facepalm: I'll just drop it then.